### PR TITLE
COMMON/UCX: suppressed coverity warnings

### PR DESCRIPTION
**Pull request by hoopoepg**
- suppressed coverity warnings - added log messages on failed calls

Signed-off-by: Sergey Oblomov <sergeyo@mellanox.com>

### CODE REVIEW

**Comment by hoopoepg** [comment]
@yosefe look please

File: ompi/mca/osc/ucx/osc_ucx_component.c [hunk 1]
```diff
@@ -825,7 +825,9 @@ int ompi_osc_ucx_free(struct ompi_win_t *win) {
         ucp_worker_progress(mca_osc_ucx_component.ucp_worker);
     }
 
-    opal_common_ucx_worker_flush(mca_osc_ucx_component.ucp_worker);
+    if (OMPI_SUCCESS != opal_common_ucx_worker_flush(mca_osc_ucx_component.ucp_worker)) {
+        OSC_UCX_VERBOSE(1, "opal_common_ucx_worker_flush failed");
```
Comment (yosefe) [inline ompi/mca/osc/ucx/osc_ucx_component.c]: pls print return code in all 3 error messages

Comment (hoopoepg) [reply ompi/mca/osc/ucx/osc_ucx_component.c]: added error codes

**Comment by yosefe** [comment]
:+1:

**Pull request merged**